From d995536a981ec1c1cc5385e2498b52d43a967789 Mon Sep 17 00:00:00 2001
From: ThePigFarmer <alanwedel19@gmail.com>
Date: Sat, 23 Mar 2024 20:44:35 -0500
Subject: [PATCH 2/2] added ToolsWindow icons

---
 agopengps.qrc       |  1 +
 qml/ToolsWindow.qml | 13 ++++++++++++-
 2 files changed, 13 insertions(+), 1 deletion(-)

diff --git a/agopengps.qrc b/agopengps.qrc
index 16bc9ee..e6f9eae 100644
--- a/agopengps.qrc
+++ b/agopengps.qrc
@@ -629,5 +629,6 @@
         <file>qml/components/ButtonColor.qml</file>
         <file>qml/components/ExpandableColumn.qml</file>
         <file>qml/components/ScrollViewExpandableColumn.qml</file>
+        <file>images/TrashContourRef.png</file>
     </qresource>
 </RCC>
diff --git a/qml/ToolsWindow.qml b/qml/ToolsWindow.qml
index ce1b9aa..db13d99 100644
--- a/qml/ToolsWindow.qml
+++ b/qml/ToolsWindow.qml
@@ -9,7 +9,6 @@ import "components"
 //    visible: false
 //    height: mainWindow.height
     Drawer {
-
         id: toolsMenu
         width: 250
         height: mainWindow.height
@@ -17,17 +16,21 @@ import "components"
 //        onVisibleChanged: if (visible === false){
 //                             toolsWindowItem.visible = false
 //                          }
+
         contentItem: Rectangle{
             id: toolsMenuContent
             anchors.fill: parent
             height: toolsMenu.height
             color: "black"
         }
+
         ScrollViewExpandableColumn {
             id: toolsGrid
             anchors.fill: parent
+
             IconButtonTextBeside {
                 id: wizards
+                icon.source: "/images/WizardWand.png"
                 width: 250
                 height: 50
                 text: qsTr("Wizards")
@@ -37,6 +40,7 @@ import "components"
 
             IconButtonTextBeside {
                 id: charts
+                icon.source: "/images/Chart.png"
                 width: 250
                 height: 50
                 text: qsTr("Charts")
@@ -46,6 +50,7 @@ import "components"
 
             IconButtonTextBeside {
                 id: smABCurve
+                icon.source: "/images/ABSmooth.png"
                 width: 250
                 height: 50
                 text: qsTr("Smooth AB Curve")
@@ -54,6 +59,7 @@ import "components"
 
             IconButtonTextBeside {
                 id: delContourPaths
+                icon.source: "/images/TrashContourRef.png"
                 width: 250
                 height: 50
                 text: qsTr("Delete Contour Paths")
@@ -62,6 +68,7 @@ import "components"
 
             IconButtonTextBeside {
                 id: delAppliedArea
+                icon.source: "/images/TrashApplied.png"
                 width: 250
                 height: 50
                 text: qsTr("Delete Applied Area")
@@ -69,13 +76,16 @@ import "components"
 
             IconButtonTextBeside {
                 id: webcam
+                icon.source: "/images/Webcam.png"
                 width: 250
                 height: 50
                 text: qsTr("WebCam")
                 visible:settings.setFeature_isWebCamOn
             }
+
             IconButtonTextBeside {
                 id: offsetFix
+                icon.source: "/images/YouTurnReverse.png" // this is horrible. This has nothing to do with YouTurnReverse.
                 width: 250
                 height: 50
                 text: qsTr("Offset Fix")
@@ -83,6 +93,7 @@ import "components"
             }
         }
     }
+
 //    Rectangle{ //this all needs to be done sometime
 //        id: wizardMenu
 //        width: childrenRect.width+10
-- 
2.34.1

